http_logs/operations/default.json

```diff
@@ -58,6 +58,17 @@
         }
       }
     },
+    {
+      "name": "default-size",
+      "operation-type": "search",
+      "index": "logs-*",
+      "body": {
+        "size": 1000,
+        "query": {
+          "match_all": {}
+        }
```

Comment:
Would recommend changing the existing `default` query to add a `size` field and make its value specifiable via a workload parameter.  I believe the default size is 10 for `match_all` and that could be set accordingly.

Reply:
I don't want to touch the `default` operation because I think that should always be equivalent to `GET <index>/_search` with no parameters, so don't want to introduce any confusion into that.

I do think it's a good idea to make size configurable for the new operation though so will add that!

Reply:
Added a workload parameter to the `default-size` operation, let me know what you think @gkamat !